assets/js/state/softwareUpdates.js

```diff
@@ -0,0 +1,44 @@
+import { createAction, createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  loading: false,
+  softwareUpdates: [],
```

Comment:
The structure of `softwareUpdates`is an array of 
```javascript
softwareUpdates: [
    { hostId: 'example_id', softwareUpdates: [ ...sumaPatches ]},
    ...
]
```

What do you think?

Reply:
we could consider an alternative where we keep a dictionary like `hostId -> discoveredSoftwareUpdates`

```
softwareUpdates: {
  "host_id": {relevant_patches: [...], upgradable_packages: [...]}
}
```
it might be helpful to a direct access by host id.